pkg/backup/util/util.go

```diff
@@ -87,6 +87,8 @@ func GenerateS3CertEnvVar(s3 *v1alpha1.S3StorageProvider) ([]corev1.EnvVar, stri
 			Value: s3.StorageClass,
 		},
 	}
+	// KMS has to use with IAM for get credentail to decryption the code
+	// so if use KMS, the secretKey and accesssKey can be ignore
 	if s3.SecretName != "" {
 		envVars = append(envVars, []corev1.EnvVar{
 			{
```

Comment:
No need to consider the case where accessKeys are encrypted by KMS?

Reply:
though use rarely use like this, we can't don't allow them do like this